src/main/resources/feedbackQuestionConstSumSubmissionFormTemplate.html

```diff
@@ -4,8 +4,9 @@
 <div id="constSumInfo-${questionIndex}-${responseIndex}" class="form-group" ${constSumOptionVisibility}>
   <div class="col-sm-6 text-center" style="clear: both;">
     <hr class="margin-top-0">
-    <p class="text-color-blue" id="constSumMessage-${questionIndex}-${responseIndex}">
-    </p>
+    <p class="text-color-blue" id="constSumMessage-${questionIndex}-${responseIndex}"></p>
+    <p class="text-color-blue" ${requireMin}>Minimum points per ${optionRecipientDisplayName} is ${constSumPointsMin}</p>
```

Comment:
You will need write JS code to dynamically update the text. If the user doesn't enter anything or the input is valid, we should hide the message.